components/homepage/Hero.tsx

```diff
@@ -136,39 +135,37 @@ const Hero = () => {
                 >
                     <Fade sx={{ position: "relative" }}>
                         <Box
+                            as="video"
                             sx={{
-                                backgroundImage: `url(https://res.cloudinary.com/guido-la-rosa/image/upload/v1657847734/carrot/Header-image_big_auuteh.png)`,
                                 width: [
                                     "100%",
                                     "calc(2.4rem * 11)",
                                     "31.2rem",
                                     "40.8rem",
                                 ],
-                                height: [
-                                    "120vw",
-                                    "calc(2.4rem * 16)",
-                                    "43.66rem",
-                                    "57rem",
-                                ],
+                                // paddingTop: "56.25%",
```

Comment:
Comment to remove.

Reply:
Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39

Reply:
Addressed in 00ab6e73c6e7d0a5fc453fc39db06032ab7b4b39